packages/config/src/ProfileInfo.ts

```diff
@@ -361,7 +364,8 @@ export class ProfileInfo {
     public mergeArgsForProfile(profile: IProfAttrs): IProfMergedArg {
         const mergedArgs: IProfMergedArg = {
             knownArgs: [],
-            missingArgs: []
+            missingArgs: [],
+            secureArgs: []
```

Comment:
In the mergeArgsForXXX functions, knownArgs and missingArgs are mutually exclusive. Every argument belongs into one of those categories. Each secureArg will fall into one of the existing categories of known or missing.

- Do you think that the consumer will want a separate list of secure arguments?   

- Will the new "isSecure" property not be sufficient for the consumer to process an argument correctly from the known or missing args?

Reply:
Both good points - `secureArgs` is likely unnecessary and can be removed.

> Do you think that the consumer will want a separate list of secure arguments?

I don't think it's a requirement, and keeping the properties of `IProfMergedArg` mutually exclusive is probably a good idea.

> Will the new "isSecure" property not be sufficient for the consumer to process an argument correctly from the known or missing args?

When testing these APIs in a VSCE, it seems simpler to handle secure args using only the `isSecure` property instead of having them in a separate array.

Reply:
Perhaps I should rename `isSecure` to `secure`? Wondering if the word `is` in the property name is unnecessary.

Reply:
I do not think there is any firm rule, but my observation is that "isXxx" tends to be the name of a function to determine some condition, but xxx is more often used as the name of a property.